### PR TITLE
Feat/guardrail view

### PR DESCRIPTION
**Pull request by KJES4**

### CODE REVIEW

**Comment by RSoulatIOHK** [comment]
Thanks Kathryn. Just a FYI, you're adding people as assignee and not reviewer, but you already did all the work! ^^

**Review by RSoulatIOHK — APPROVED** [review]
Not sure what the scope of the PR is supposed to be but with the fix of the bug we found, it looks good from my end

File: frontend/src/components/MoreDetailsDrawer.tsx [hunk 1]
```diff
@@ -21,40 +22,77 @@ export default function MoreDetailsDrawer() {
...
    
   const getRowDetails = () => {
-    if (!validationResults || !selectedRowName) {
+    if (!validationResults || !selectedRowName || selectedRowName.length < 1) {
       return null;
     }
 
-    const selectedRowArray = selectedRowName.split(/[.[\]]+/).filter(k => k);
     let rowDetails: any = validationResults;
+    console.log(rowDetails);
```
Comment (EehMauro) [inline frontend/src/components/MoreDetailsDrawer.tsx]: We should remove this console.log right?

Comment (KJES4) [reply frontend/src/components/MoreDetailsDrawer.tsx]: Yes, thanks for catching that! I've just removed it and pushed the changes.

**Review by EehMauro — APPROVED** [review]

**Pull request merged**